apps/server/src/modules/learnroom/uc/course-copy.uc.ts

```diff
@@ -34,6 +35,7 @@ export class CourseCopyUC {
 		const user = await this.authorisation.getUserWithPermissions(userId);
 		const originalCourse = await this.courseRepo.findById(courseId);
 		let originalBoard = await this.boardRepo.findByCourseId(courseId);
+		this.featureEnabled();
```

Comment:
I would put this as the first thing in the method. If feature is not enabled in the first place, there is no point in making other requests, check for permissions, etc....